crates/bevy_ui/src/flex/mod.rs

```diff
@@ -172,6 +172,14 @@ without UI components as a child of an entity with UI components, results may be
         }
     }
 
+    pub fn remove_entities(&mut self, entities: impl Iterator<Item = Entity>) {
```

Comment:
Nit: I think this would be a bit clearer and more idiomatic if this took an `IntoIterator`, and then converted it inside this method.